### ISSUE TITLE
GB radii value error

### ISSUE THREAD

**Issue by isjoung** [pandegroup/openmm#1488]
I turned on GBn2 and tried to load a system, but it failed with this error:
```
ValueError: Radii must be between 1 and 2 Angstroms for neck lookup
```
I tried to trace back to the source of the problem. Some radii of hydrogen atoms in ARG have 1.170 A according to mbondi3 radii set. Since the offset is 0.195141 A, the value drops below 1 A after the subtraction of the offset. (1.170-0.195141 = 0.974859)

I have not clearly understood the exact meaning of the error message, but is it really a problem and should an error be raised?

**Comment by jchodera** [comment]
Pinging @swails, who might have insight into this (since he last refreshed that code).

**Comment by peastman** [comment]
I think he just fixed this error a couple of days ago: #1484.

**Comment by swails** [comment]
Indeed, this is fixed by #1484

**Comment by isjoung** [comment]
Looks good!
